.github/workflows/build-preflight-in-a-box.yml

```diff
@@ -0,0 +1,107 @@
+name: New Tag
+on: push
+
+jobs:
+  build:
+    # This job only runs for issue comments
+    name: Update package
+    runs-on: ubuntu-latest
+    #     permissions:
+    #       contents: read
+    #       packages: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set Node.js version 16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - name: Install deps and build (with cache)
+        uses: bahmutov/npm-install@v1
+
+        # 'TODO: get the ulr to pass to the sandbox
+
+        # # Use the sandbox to
+        # 1. clone the repo
+        # 2. install dependencies of the repo
+        # 3. test preflight inside
+      - name: build image
```

Comment:
Naming here - this step is not building the image, it's 

If it will also build the image in the final script, then we can say something like "Compile TypeScript and build Docker image